iDiff/differs/differs.go

```diff
@@ -8,41 +8,41 @@ import (
 	"github.com/GoogleCloudPlatform/runtimes-common/iDiff/utils"
 )
 
-var diffs = map[string]func(string, string, bool) (string, error){
+var diffs = map[string]func(string, string, bool, bool) (string, error){
 	"hist": History,
 	"dir":  Package,
 	"apt":  AptDiff,
 	"node": NodeDiff,
 	"pip":  PipDiff,
 }
 
-func Diff(arg1, arg2, differ string, json bool) (string, error) {
```

Comment:
You could also extract the json encoding logic from the differs.  It gets duplicated for every differ, and it should be independent from the "diffing".  If a particular differ needs to encode json a special way, you might consider returning a more structured `ImageDiffOutput` struct that can json encode itself.